app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -38,14 +75,20 @@ class OptionsContainer extends React.Component {
       answer.id = this.props.answer.id;
       this.props.updateAnswer(answer.building_id, answer);
     }
+
+    setTimeout(() => this.focusFirstDependent(), 500);
   }
 
   render() {
     const optionProps = {
       options: this.props.options,
       answer: this.props.answer,
-      onSelect: this.handleSelect.bind(this)
+      onSelect: this.handleSelect.bind(this),
```

Comment:
I would either bind this in the constructor or use the rocket methods like you've already been doing